add compile instructions for vs6

PiecePaperCode opened the pull request:
## Changes

A Markdown file of instructions to compile with visual studio code 6.
It uses the WIP pull request as its basis: https://github.com/TheSuperHackers/GeneralsGameCode/pull/365.
It uses the portable version of vs6 from here: https://github.com/itsmattkc/MSVC600.
You need to have vs6, cmake and git on your system to by installed manualy by user. 

Rendered View Here: https://github.com/PiecePaperCode/GeneralsWiki/blob/main/SourceCode/Builds/vs6_compile.md

real2nabeel commented on Code/exe/compile_vs6.md:
```diff
@@ -0,0 +1,121 @@
+
+Execute the setup build environment script. In your cmd type this.
+
+`"C:\Programm Files (x86)\Microsoft Visual Studio\VC98\Bin\VCVARS32.bat"`
```
```suggestion
`"C:\Program Files (x86)\Microsoft Visual Studio\VC98\Bin\VCVARS32.bat"`
```

real2nabeel commented on Code/exe/compile_vs6.md:
```diff
@@ -0,0 +1,121 @@
+
+**Compilation**
+
+`cmake -preset vc6`
```
```suggestion
`cmake --preset vc6`
```

real2nabeel commented on Code/exe/compile_vs6.md:
```diff
@@ -0,0 +1,121 @@
+
+`cd CnC_Generals_Zero_Hour`
+
+Copy the files from Directx 8 contained in the folder `Lib` and `Include` to the `Dependencies\DX8SDK`. Just the file contents of `Lib` and `Include` into the `DX8SDK` folder. 
```
To improve clarity so people don't do this mistake
```suggestion
Copy all files from the `lib` and `include` folders inside the `DirectX 8` directory and place them directly into `Dependencies\DX8SDK` —do not keep them inside lib or include. Only the file contents should be in DX8SDK, not the `lib` and `include` folders themselves.
```

real2nabeel commented on Code/exe/compile_vs6.md:
```diff
@@ -0,0 +1,121 @@
+- **Dont check "Setup Environment Variables" checkbox
+- Restart your computer
+- Install MSDN a window will pop up
+- Try to install it but it worked if it failed
```
```suggestion
- Try to install, if it fails or you cannot find a CD path, ignore it.
```

DevGeniusCode commented on Builds/vs6_compile.md:
```diff
@@ -0,0 +1,121 @@
+# Compile Command & Conquer Zero Hour 
+**with VS6 and VC98**
```
The seem that title broken into two lines.

PiecePaperCode replied:
ill remove this

DevGeniusCode commented on Builds/vs6_compile.md:
```diff
@@ -0,0 +1,121 @@
+
+## Clone
+
+Clone the code from richard
+
+`git clone https://github.com/richardboegli/CnC_Generals_Zero_Hour.git`
```
Maybe it's worth linking to xezon's fork, which is very clean and will later be merged into TheSuperHackers' fork?

PiecePaperCode replied:
Yes i will premilirary point to SuperHackers code now as i expect this to be merged very soon. I used OmniBlade fork wich is currently as PR WIP https://github.com/TheSuperHackers/GeneralsGameCode/pull/365

PiecePaperCode commented:
- I have updated the path of the markdown file to the new source code build folder
- I have changed the instructions to point to https://github.com/TheSuperHackers/GeneralsGameCode now.
- I have added troubleshooting steps.

DevGeniusCode commented on SourceCode/Builds/vs6_compile.md:
```diff
@@ -0,0 +1,127 @@
+
+## Clone
+
+Clone the code from TheSuperHackers
+
+`git clone https://github.com/TheSuperHackers/GeneralsGameCode.git`
+
+Open `cmd.exe` and navigate to the cloned source code.
+
+`cd GeneralsGameCode`
```
Duplication of prerequisites?

DevGeniusCode commented on SourceCode/Builds/vs6_compile.md:
```diff
@@ -0,0 +1,127 @@
+
+Installation of tools and software that are needed for compilation.
+
+**Install Visual Studio 6**
+
+- Download the portable Visual Studio 6 as ZIP file from GitHub.
+- Extract the contents and put them in the default install folder.
+- Installation folder: `C:\Program Files (x86)\Microsoft Visual Studio\`
```
Maybe should add something like
Alternatively you can install the VC6 setup file

DevGeniusCode commented on SourceCode/Builds/vs6_compile.md:
```diff
@@ -0,0 +1,127 @@
+Download the following binary and software and have them ready in a project folder.
+
+- Visual Studio 6.0 Portable [Link](https://github.com/itsmattkc/MSVC600)
+- Source code [Link](https://github.com/TheSuperHackers/GeneralsGameCode)
```
It should be noted that since this is a WIP, it currently only exists for ZeroHour, and that it has not yet been merged and therefore must be downloaded from OmniBlade.

PiecePaperCode replied:
add a note its a WIP

DevGeniusCode commented on SourceCode/Builds/vs6_compile.md:
```diff
@@ -0,0 +1,127 @@
+# Compile Command & Conquer Zero Hour 
```
Could you change the file name to something like `build_with_msvc6`, and also indicate in the title that it is VC6?

PiecePaperCode replied:
file name changed

xmesaj2 commented on SourceCode/Builds/build_with_msvc6.md:
```diff
@@ -0,0 +1,141 @@
+
+`git clone https://github.com/OmniBlade/CnC_Generals_Zero_Hour.git`
+
+`cd CnC_Generals_Zero_Hour.git`
```
`.git` here is not part of the directory name
